src/Optimization/hiopHessianLowRank.hpp

```diff
@@ -124,6 +124,7 @@ class hiopHessianLowRank : public hiopMatrix
   virtual void timesVecCmn(double beta, hiopVector& y, double alpha, const hiopVector&x, bool addLogBarTerm = false) const;
 
 protected:
+  friend class hiopAlgFilterIPMQuasiNewton;  
   int l_max; //max memory size
   int l_curr; //number of pairs currently stored
```

Comment:
All member variable names should end with trailing underscore to distingusish them from local variables.
Well, seems we need to change many things here and there. Maybe a new PR?

Reply:
yes